src/main/java/jenkins/plugins/openstack/compute/SlaveOptions.java

```diff
@@ -270,6 +295,13 @@ private Object readResolve() {
         return base;
     }
 
+    /** Returns null if our <tt>base</tt> value is the same as the <tt>def</tt>ault value. */
+    private @CheckForNull <T extends Collection<?>> T _eraseC(@CheckForNull T base, @CheckForNull T def) {
+        if (def == null || def.isEmpty()) return base;
```

Comment:
Ditto, we do not need these override I think.

Reply:
Done in https://github.com/jenkinsci/openstack-cloud-plugin/pull/270/commits/5456d1d835b8da5588b9f7e3ce96730cc0b1a071